rental.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'person'
+require_relative 'book'
+
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+
+    person.add_rental(self)
+    book.add_rental(self)
```

Comment:
in the initializer all we have to do is to add the new created Rental to person rentals and book rentals to insure that it is added both sides 

````ruby
@book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
````
and in this case no need to to push from the book  and person add_rental method creating a new Rental instance will push the rental in both sides